roofit/hs3/src/JSONIOUtils.cxx

```diff
@@ -0,0 +1,6 @@
+#include "JSONIOUtils.h"
+
+bool endsWith(const std::string_view str, const std::string_view suffix)
```

Comment:
```suggestion
bool endsWith(std::string_view str, std::string_view suffix)
```
This doesn't need to be `const`. String views are passed by value, so it doesn't matter to the caller if they are changed or not. The `const` is usually used only for pointers and references, such that the function doesn't have side effects on the variables held by the caller.